build.sh

```diff
@@ -307,11 +307,47 @@ EOT
           --cov wdae \
           wdae || true'
 
-    build_run_container coverage combine dae/.coverage wdae/.coverage
+    build_run_container cp ./results/wdae-junit.xml ./test-results/
+  }
+
+  build_stage "Tests - tests"
```

Comment:
I don't understand, why do we need this stage? Don't we already do all of that?

Reply:
I've added some comments. Hope this helps